*** Starting uWSGI 2.0.18 (64bit) on [Wed Aug 28 19:03:08 2019] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-36) on 07 August 2019 10:54:11
os: Linux-3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /usr/project/firstproject
writing pidfile to /usr/project/firstproject/uwsgi/uwsgi.pid
detected binary path: /root/anaconda3/envs/py3/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /usr/project/firstproject
your processes number limit is 3795
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 192.168.1.128 fd 6
probably another instance of uWSGI is running on the same address (:9090).
bind(): Address already in use [core/socket.c line 769]
VACUUM: pidfile removed.
unlink(): No such file or directory [core/uwsgi.c line 1673]
*** Starting uWSGI 2.0.18 (64bit) on [Wed Aug 28 19:14:35 2019] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-36) on 07 August 2019 10:54:11
os: Linux-3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /usr/project/firstproject
writing pidfile to /usr/project/firstproject/uwsgi/uwsgi.pid
detected binary path: /root/anaconda3/envs/py3/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /usr/project/firstproject
your processes number limit is 3795
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 192.168.1.128 fd 6
probably another instance of uWSGI is running on the same address (:9090).
bind(): Address already in use [core/socket.c line 769]
VACUUM: pidfile removed.
unlink(): No such file or directory [core/uwsgi.c line 1673]
